indra/newview/skins/default/xui/en/menu_url_slurl.xml

```diff
@@ -27,6 +27,15 @@
     </menu_item_call>
     <menu_item_separator
      layout="topleft" />
+    <menu_item_call
+     label="Create Landmark for SLurl"
+     layout="topleft"
+     name="landmark">
+        <menu_item_call.on_click
+         function="Url.CreateLandmark" />
+        <menu_item_call.on_enable
```

Comment:
It's better to hide new element instead of simply disabling it. Users will be confused why element is disabled and will look for a way to enable it, better for it to not be seen untill feature is deployed server side.

Reply:
I think the same way, and I started from the attribute is_visible, but it is not supported here

Reply:
menu_item_call.on_visible is used just fine in other places, how can it be not supported?